src/include/processor/operator/persistent/copy_rdf.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "processor/operator/sink.h"
+
+namespace kuzu {
+namespace processor {
+
+struct CopyRdfSharedState {
+    std::shared_ptr<FactorizedTable> fTable;
+};
+
+class CopyRdf : public Sink {
```

Comment:
Consider making this operator as a general CopySink op. The pipeline of copy rel is facing the same problem that there are three pipelines running, and the last pipeline is serving as the dummy CopySink now to flush information to the final shared fTable, but can be changed to use `CopySink` as the dummy final op to flush information out.

Reply:
As discussed we will do this in a different PR